README.md

```diff
@@ -0,0 +1,151 @@
+# aws-cli
+
+[![Build Status](https://travis-ci.org/aws/aws-cli.svg?branch=develop)](https://travis-ci.org/aws/aws-cli)
+
+[![Gitter](https://badges.gitter.im/aws/aws-cli.svg)](https://gitter.im/aws/aws-cli)
+
+This package provides a unified command line interface to Amazon Web Services.
+
+Jump to:
+
+- [Getting Started](#getting-started)
+- [Getting Help](#getting-help)
+- [More Resources](#more-resources)
+
+## Getting Started
+
+### Requirements
+
+The aws-cli package works on Python versions:
+
+- 2.7.x and greater
+- 3.4.x and greater
+- 3.5.x and greater
+- 3.6.x and greater
+- 3.7.x and greater
+- 3.8.x and greater
+
+On 10/09/2019 support for Python 2.6 and Python 3.3 was deprecated and support was dropped on 01/10/2020. To avoid disruption, customers using the AWS CLI on Python 2.6 or 3.3 will need to upgrade their version of Python or pin the version of the AWS CLI in use prior to 01/10/2020. For more information, see this [blog post](https://aws.amazon.com/blogs/developer/deprecation-of-python-2-6-and-python-3-3-in-botocore-boto3-and-the-aws-cli/).
+
+*Attention!*
+
+*We recommend that all customers regularly monitor the [Amazon Web Services Security Bulletins website](https://aws.amazon.com/security/security-bulletins) for any important security bulletins related to aws-cli.*
+
+### Installation
+
+The safest way to install aws-cli is to use [pip](https://pip.pypa.io/en/stable/) in a `virtualenv`:
+
+    $ python -m pip install awscli
+
+or, if you are not installing in a `virtualenv`, to install globally:
+
+    $ sudo python -m pip install awscli
+
+or for your user:
+
+    $ python -m pip install --user awscli
+
+If you have the aws-cli installed and want to upgrade to the latest version you can run:
+
+    $ python -m pip install --upgrade awscli
+
+Note
+
+On macOS, if you see an error regarding the version of `six` that came with `distutils` in El Capitan, use the `--ignore-installed` option:
+
+    $ sudo python -m pip install awscli --ignore-installed six
+
+This will install the aws-cli package as well as all dependencies. You can also [download the tarball](https://pypi.org/project/awscli/). Once you have the aws-cli directory structure on your workstation, you can run:
+
+    $ cd <path_to_awscli>
+    $ python setup.py install
```

Comment:
Let's remove this example on how to install the sdist. I don't think it is really needed. Also I'm not sure if I saw this anywhere but it would be good to link to the other installer guides for the bundled installer and MSI.